development/issue_tracking.md

```diff
@@ -19,6 +19,12 @@ When opening an issue, do not check any boxes in the *Status* section of the iss
 3. Once the triage team has approved the issue, they will add both a *triage/approved* label and assign a developer.
 4. Once a defect has been fixed and closed, the triage team will add a new community member as assignee to perform validation on the appropriate branch.
 
+#### Triage Queries
+* [Open Issues: Without Labels](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aissue+user%3Aopensecuritycontroller+no%3Alabel+is%3Aopen+)  
```

Comment:
This are some ideas for initial queries. I am sure we will come up with other queries as the process evolves.